lib/ansible/modules/files/lineinfile.py

```diff
@@ -82,6 +82,8 @@
       - If the first match is required, use(firstmatch=yes).
       - A special value is available; C(EOF) for inserting the line at the end of the file.
       - If specified regular expression has no matches, EOF will be used instead.
+      - Cannot set with C(insertbefore).
+      - If C(insertbefore) is set, default value C(EOF) will be ignored.
       - If regular expressions are passed to both C(regexp) and C(insertafter), C(insertafter) is only honored if no match for C(regexp) is found.
       - May not be used with C(backrefs).
```

Comment:
Combine line 85 with this line: 
```suggestion
       - May not be used with C(backrefs) or C(insertbefore).
```

Reply:
Thanks for reply. I fixed that. Could you check my change?